### ISSUE TITLE
Is it safe to delete files from /var/tmp?

### ISSUE THREAD

**Issue by LemonSpike** [LemonSpike/TIL#2]
> I encountered this folder when producing a Wi-Fi diagnostics report.
> The report was saved in /var/tmp and I wanted to delete the file.
>
> I deleted the files because they do not seem critical to the OS functionality, 
> and then saw other files in the folder related to `configd` such as `configd-state`,
> `configd-store.plist` and `configd-pattern.plist`.
>
> I did not delete these files because `configd` sounds important xD. More details
> about `configd` [here](https://github.com/LemonSpike/TIL/issues/1).
>
> I then [Stack'd](https://askubuntu.com/questions/864383/is-it-safe-to-delete-files-from-var-tmp).
> This gave me the answer because I didn't know where to find the documentation for /var/tmp.
>
> However, there are UNIX filesystem resources such as [Imperial](https://www.doc.ic.ac.uk/~wjk/UnixIntro/Lecture2.html).
> This resource however didn't include a description for the `var` root folder.
> I hypothesised that this may be a Mac-only root folder but I was unsure because macOS is derived from
> UNIX largely.
> The authoritative source for UNIX FSs is [The Filesystem Hierarchy Standard](https://www.pathname.com/fhs/pub/fhs-2.3.html#THEVARHIERARCHY):
> /var contains variable data files. This includes spool directories and files, administrative and logging data, and transient and temporary files.
>
> [LINFO](http://www.linfo.org/var.html) was a good resource:
> /var is a standard subdirectory of the root directory in Linux and other Unix-like operating systems
> that contains files to which the system writes data during the course of its operation.
>
> TL;DR: It doesn't seem too important 👍🏾. 
> The system works first time you boot it, without `configd`.
> `configd` is just some app bundle settings, and it's only stored random diagnostic reports.
>
> There was also a _socket_ file (/var/tmp/filesystemui.socket).
> The file type is actually socket:
> $ file /var/tmp/filesystemui.socket
> /var/tmp/filesystemui.socket: socket
>
> More info on sockets [here](https://en.wikipedia.org/wiki/WebSocket).
>
> Next time, use the Filesystem Hierarchy Standard to search about UNIX folders 😀.
> Don't Duck out straight away 👎🏾 
> Learnt for next time to review the whole problem first before jumping to Duck 😄.
